File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int> ans(nums.size());
        
        
         for(int i = 0; i < nums.size(); i++){
            ans[i]= i == 0? nums[i] : ans[i-1] +nums[i];
        }
        
        
     return ans;   
    }
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        
        //ArrayList<Integer> arr = new ArrayList<>();
        int arr[] = new int[nums.length];
        int sum =0;
        int curr_sum=0;
        
        for(int i = 0; i < nums.length; i++){
            sum += nums[i];
            //arr.add(sum);
            arr[i]=sum;
        }
        
        // for(int i =nums.length-2; i>=0;i--){
        //     curr_sum =
        // }
        return arr;
    }
}